### ISSUE TITLE
Django 3 incompatibility

### ISSUE THREAD

**Issue by karimbahgat** [makinacorpus/django-leaflet#332]
I recently encountered the following error when trying to push a Django app with django-leaflet (v0.26.0) to Heroku:

```
Traceback (most recent call last):
         File "/app/.heroku/python/lib/python3.9/site-packages/leaflet/__init__.py", line 30, in <module>
           import six
       ModuleNotFoundError: No module named 'six'
       During handling of the above exception, another exception occurred:
       Traceback (most recent call last):
         File "/tmp/build_c040aa9d/manage.py", line 21, in <module>
           main()
         File "/tmp/build_c040aa9d/manage.py", line 17, in main
           execute_from_command_line(sys.argv)
         File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
           utility.execute()
         File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
           django.setup()
         File "/app/.heroku/python/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
           apps.populate(settings.INSTALLED_APPS)
         File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/registry.py", line 91, in populate
           app_config = AppConfig.create(entry)
         File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/config.py", line 224, in create
           import_module(entry)
         File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
           return _bootstrap._gcd_import(name[level:], package, level)
         File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
         File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
         File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
         File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
         File "<frozen importlib._bootstrap_external>", line 855, in exec_module
         File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
         File "/app/.heroku/python/lib/python3.9/site-packages/leaflet/__init__.py", line 32, in <module>
           from django.utils import six
       ImportError: cannot import name 'six' from 'django.utils' (/app/.heroku/python/lib/python3.9/site-packages/django/utils/__init__.py)
 !     Error while running '$ python manage.py collectstatic --noinput'.
```

The key error here seems to be the assumption that `django.utils` contains a reference to six (`from django.utils import six`), but this [appears to have been removed as of Django 3](https://stackoverflow.com/questions/59193514/importerror-cannot-import-name-six-from-django-utils). 

The fix for my situation was to specify `django==2`, but in general I guess this means that django-leaflet won't work for Django 2? I believe a similar issue was raised in #265, but this error seems to originate from a different line of code.. 

Not sure what's most preferable: 
1. to find some solution for the six package dependency? 
2. or specify that django-leaflet requires `django<3` in the `requires` field of `setup.py`? 

I'd be happy to do a PR as long as I know the preferred solution.

**Comment by karimbahgat** [comment]
I'm using 0.26 (Dec 2019) because of a change that was introduced in 0.27 that necessitates having a GDAL installation. Personally I don't need a solution, just thought i'd raise the issue. Has it already been fixed in 0.27 or 0.28?

**Comment by claudep** [comment]
Oh, I see. However the longer you will stay on a legacy version, the more you will encounter such compatibility issues. Either make the effort to install GDAL, or maybe try to suggest a PR to re-support django-leaflet without GDAL (I'm not a maintainer, so no guarantee of acceptation, of course).
..

**Comment by Gagaro** [comment]
I think GDAL is required because we import some fields from django which need GDAL. If you find a way to fix that, feel free to open a PR.

**Comment by claudep** [comment]
There used to be some fallbacks in the past for systems without GEOS/GDAL, this was removed in d7f7b8f29a289ff. I don't know if it's possible to reintroduce it with current code, or even if this is desirable. Maybe a design choice from maintainers…

**Comment by Gagaro** [comment]
I think it was due to it not really working with last Django versions. Did you try to add it back without GDAL installed?

**Comment by karimbahgat** [comment]
I was actually planning on raising an issue/PR regarding the new GDAL dependency, which I definitely think would be desirable, so would be happy to continue that conversation. I confirmed again that the newest django-leaflet 0.28 without GDAL fails because it cannot find gdal. 

As for this issue regarding Django version/six incompatibility, I did some more digging and found that [the problem was indeed fixed as of May 18, 2020](https://github.com/makinacorpus/django-leaflet/commit/844887affe607d3f115920c862f8ea2b45e19ed8#diff-2824472c3107ae159579218da1fea3e94709620ba016f95ec8d5e9dd38101583). So the issue should be fixed in later versions, closing this issue.
